sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetFilterSuite.scala

```diff
@@ -544,7 +544,7 @@ class ParquetFilterSuite extends QueryTest with ParquetTest with SharedSQLContex
     }
   }
 
-  test("Do not push down filters incorrectly when inner name and outer name are the same") {
+  test("SPARK-16371 Do not push down filters when inner name and outer name are the same") {
```

Comment:
Hi @rxin, I am sorry for asking such a question but I couldn't find the documentation for this.

I am sometimes confused if I should add the JIRA number as prefix of a test or not. I am trying to comply this rule which I think, writing single test requires to add the JIRA number in the test name and writing multiple tests does not.

Is this just encouraged to write this always? If so, I will try to comply this from now on and will help you leave such comments when I happened to look at some other PRs.

(I don't intend to change this across codebase)